scripts/weather.js

```diff
@@ -1,4 +1,38 @@
-const atlWeather = {"coord":{"lon":-84.39,"lat":33.75},"weather":[{"id":800,"main":"Clear","description":"clear sky","icon":"01d"}],"base":"stations","main":{"temp":272.76,"pressure":1022,"humidity":40,"temp_min":270.15,"temp_max":275.37},"visibility":16093,"wind":{"speed":3.1,"deg":340},"clouds":{"all":1},"dt":1551797654,"sys":{"type":1,"id":4399,"message":0.0072,"country":"US","sunrise":1551787231,"sunset":1551829066},"id":4180439,"name":"Atlanta","cod":200};
+const atlWeather = {
```

Comment:
Nice. It's always a good idea to reformat so that you get a clearer picture of the data you're working with!